code/__DEFINES/traits.dm

```diff
@@ -542,7 +542,9 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define HIGHLANDER_TRAIT "highlander"
 
 ///generic atom traits
-#define DO_NOT_SPLASH "do_not_splash"
+/// Trait from [/datum/component/rust]. Its rusty and should be applying a special overlay to denote this.
+#define TRAIT_ATOM_RUSTY "rust_trait"
```

Comment:
Don't rename them to `_ATOM_`, it was fine before--this is an implementation detail.

Reply:
Alright